roles/hsm_prep/README.md

```diff
@@ -0,0 +1,66 @@
+# HSM Role
+
+In order to use HSMs, the barbican images need to be customized to include the HSM software.  For now, this is something
+that we expect customers to do with scripts that we will provide as part of the barbican-operator code.
+
+The purpose of this role is to:
+* Generate new images for the barbican-api and barbican-worker containing the HSM software
+* Upload those images to a private repository for use in setting up a CI job.
+* Create any required config to be mounted by the barbican images to interact with the HSM
+
+For the Lunasa, we expect some preparatory steps to be completed prior to execution in order for the
+role to complete successfully.
+* We expect a VM that contains the Lunasa client software and that is registered as a lunasa client.
+  This VM should contain the following contents:
+  * The contents of the minimal linux client in a zipped tar file.
+  * The lunasa binaries that need to be added to the image under a specified directory.
+  * The lunasa HSM server cert.
```

Comment:
Since you're going to tar up the linux-minimal client, why not also add the binaries there too?

Reply:
yeah - I suppose we could do that.  six and a half dozen ..